boards/arm/mec1501modular_assy6885/doc/index.rst

```diff
@@ -0,0 +1,255 @@
+- FAN PWM interface
+- JTAG/SWD, ETM and MCHP Trace ports
+- PECI interface 3.0
+- I2C voltage translator
+- 10 SMBUS headers
+- 4 SGPIO headers
+- VCI interface
+- 5 independent Hardware Driven PS/2 Ports
+- eSPI header
+- 3 Breathing/Blinking LEDs
+- 2 Sockets for SPI NOR chips
+- One reset and VCC_PWRDGD pushbuttons
+- One external PCA9555 I/O port with jumper selectable I2C address.
+- One external LTC2489 delta-sigma ADC with jumper selectable I2C address.
+- Board power jumper selectable from +5V 2.1mm/5.5mm barrel connector or USB Micro A connector.
+
+For more information about the SOC please see the `MEC1501 Reference Manual`_
+
+Supported Features
+==================
+
+The mec1501modular_assy6885 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| ESPI      | on-chip    | espi                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
+
+The default configuration can be found in the
+:zephyr_file:`boards/arm/mec1501modular_assy6885/mec1501modular_assy6885`
+Kconfig file.
+
+Connections and IOs
+===================
+
+This evaluation board kit is comprised of the following HW blocks:
+
+- MEC1501 Modular Card ASSY 6885 Rev A0 `MEC1501 Modular EC Card - Assy_6885 Rev A0p1`
+
+System Clock
+============
+
+The MEC1501 MCU is configured to use the 48Mhz internal oscillator with the
+on-chip PLL to generate a resulting EC clock rate of 12 MHz. See Processor clock
+control register in chapter 4 "4.0 POWER, CLOCKS, and RESETS" of the data sheet in
+the references at the end of this document.
+
+Serial Port
+===========
+UART1 is configured for serial logs.
+
+Jumper settings
+***************
+
+Please follow the jumper settings below to properly demo this
+board. Advanced users may deviate from this recommendation.
+
+Jumper setting for MEC1501 Modular Assy 6885 Rev A1p0
+=================================================
```

Comment:
The title underline must be at least as long as the title, so add some more ``===``s